fenix/app/src/main/java/org/mozilla/fenix/trackingprotection/TrackingProtectionCategoryItem.kt

```diff
@@ -29,6 +29,7 @@ class TrackingProtectionCategoryItem @JvmOverloads constructor(
             defStyleAttr,
             0,
         ) {
+            binding.root.isClickable = false
```

Comment:
The `clickable` property [is explicitly set in xml](https://searchfox.org/mozilla-mobile/rev/2539bc45c927cd8ec585928216685141b6e7ec25/firefox-android/fenix/app/src/main/res/layout/fragment_tracking_protection_blocking.xml#31) so if we don't want this behavior that seems like a better place to disable it.

Tried searching for why that was made clickable in the first place but I could not find any related functionality so we could remove the click functionality. Maybe though in the next Nightly version to allow for more testing time.